src/components/pages/NewProject/components/Client.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
```

Comment:
All of these components should be in their own folder in the src/components folder

Reply:
These are bricks that form up this form, placing them within a same folder would be much more easy to find and maintain, **to be discuss**